pages.pt_BR/linux/pwd.md

```diff
@@ -0,0 +1,12 @@
+# pwd
+
+> Informa o diretório atual.
+> Mais informações: <https://man7.org/linux/man-pages/man1/pwd.1.html>.
+
+- Exibe o diretório lógico atual:
+
+`pwd -L`
```

Comment:
We only accept translation from the main (English) pages to keep easier to maintain. Could you please review this?

Reply:
@IranNeto if you want to add the `pwd -L` example, please also add it to the English page